industrial_ci/src/docker.sh

```diff
@@ -162,7 +162,8 @@ RUN echo "deb ${ROS_REPOSITORY_PATH} $UBUNTU_OS_CODE_NAME main" > /etc/apt/sourc
 RUN apt-key adv --keyserver "${APTKEY_STORE_SKS}" --recv-key "${HASHKEY_SKS}" \
     || { wget "${APTKEY_STORE_HTTPS}" -O - | sudo apt-key add -; }
 
-RUN apt-get update -qq \
+RUN sed -i "/^# deb.*multiverse/ s/^# //" /etc/apt/sources.list \
```

Comment:
Yes, it is not included for some reason, but is recommended: http://wiki.ros.org/kinetic/Installation/Ubuntu#kinetic.2BAC8-Installation.2BAC8-Sources.Configure_your_Ubuntu_repositories